code/client/clrcore/External/Blip.cs

```diff
@@ -227,7 +227,136 @@ public enum BlipSprite
 		Stopwatch = 430,
 		DollarSignCircled,
 		Crosshair2,
-		DollarSignSquared = 434
+		DollarSignSquared = 434,
+		RaceStunt,
+		HotProperty,
+		UrbanWarfare,
+		Crown = 439,
```

Comment:
For new entries, instead of/in addition to 'made up names' as in SHVDN, I'd prefer the original names as seen in `hud.gfx` be used, these are also on https://docs.fivem.net/docs/game-references/blips/

Reply:
all right i'll take a look at it!